flutter/chat_sub.dart

```diff
@@ -0,0 +1,6 @@
+channel
+  .presence
+  .subscribe(action: PresenceAction.enter)
+  .listen((member) => ui.showMemberEntered(member));
+
+channel.subscribe().listen((message) => appendToConversation(message.data));
```

Comment:
Is this missing `ui.`?

Reply:
As Flutter is a UI centric framework, developers would by default be in the context of a rendering widget. In my opinion, this would be most appropriate.